server/webpa.go

```diff
@@ -208,6 +213,16 @@ type WebPA struct {
 	// is empty, no pprof server is started.
 	Pprof Basic
 
+	//Metrics describes the metrics provider server for this application
+	Metrics Basic
+
+	//The following two fields are expected to be defined by client. Zero value is ok.
+	//Gatherer defines the metrics gathering behavior.
+	Gatherer stdprometheus.Gatherer
+
+	//Opts constitutes prometheus handling options.
+	Opts *promhttp.HandlerOpts
```

Comment:
Make this a `promhttp.HandlerOpts`, instead of a pointer.  That way, there are always default options supplied with the `WebPA` structure.